pshtt/cli.py

```diff
@@ -41,7 +41,7 @@ def main():
     if args['INPUT'][0].endswith(".csv"):
         domains = utils.load_domains(args['INPUT'][0])
     else:
-        domains = args['INPUT']
+        domains = utils.format_domains(args['INPUT'])
```

Comment:
I think it would be cleaner to make one call to `utils.format_domains()`, outside of this conditional, rather than calling it in one way for directly input domains and calling the `utils.format_domain()` function directly for domains loaded in via CSV.

This would also allow it to be one `utils.format_domains()` function, rather than two very similarly named functions.